skid_steer_velocity_controller:
  type: pandora_motor/SkidSteerVelocityController
  left_front_wheel: left_front_wheel_joint
  right_front_wheel: right_front_wheel_joint
  left_rear_wheel: left_rear_wheel_joint
  right_rear_wheel: right_rear_wheel_joint
  publish_rate: 50
  terrain:1.2

   has_velocity_limits: false
   max_velocity: 5500
   min_velocity: -5500
 
 
  